radiance_test.go

```diff
@@ -132,7 +132,7 @@ func TestNewRadiance(t *testing.T) {
 		r, err := NewRadiance()
 		assert.NoError(t, err)
 		assert.NotNil(t, r)
-		assert.NotNil(t, r.confHandler)
+		//assert.NotNil(t, r.confHandler)
```

Comment:
Is this supposed to remain commented out like this?  If so, could you provide a comment explaining why?  Alternatively, if this part of the test doesn't make sense anymore, we should just remove it.

Reply:
I'll add a line explaining, but basically, currently we're not fetching the configuration from the backend and not even creating the config handler when starting the application, so that was breaking the tests

Reply:
Fixed!